app/models/membership_application.rb

```diff
@@ -41,6 +41,9 @@ class MembershipApplication < ApplicationRecord
 
   scope :open, -> { where.not(state: [:accepted, :rejected]) }
 
+  delegate :full_name, to: :user, prefix: true, allow_nil: true
+  delegate :email, to: :user, prefix: true, allow_nil: true
+
```

Comment:
After talking with Robert after our dev team meeting on Thursday, I agree with you both that we should use `delegate :full_name, to :user ...`